helm-chart/README.md

```diff
@@ -34,13 +34,36 @@ connection:
 ```
 
 The data in the Secret object should look like this:
+
 ```yaml
 data:
   username: base64encodedPassword
 ```
+
 where `username` is the user that the Connector will use to connect to the
 database. By default the *'postgres'* user is used, as set in `.Values.connection.user`.
 
+OR 
+
+You can use db uri to connect to TimescaleDB is stored in a
+Kubernetes Secret created before the chart is deployed.
+You can set the secret name by modifying the  `connection.dburi.secretTemplate` value.
+Templating is supported and you can use:
+```yaml
+connection:
+  dburi:
+    name: db-uri
```

Comment:
do we really need this configurable?

Reply:
I don't see a real need. Should we hardcode it? As we definitely need some key in secret to refer to the value as the secret is a key/vale pair.